client_with_ssl.js

```diff
@@ -229,20 +206,60 @@ async function selectCountriesAndCities(hzClient) {
     console.log("--------------------");
 }
 
+async function nonStopMapExample(map) {
+    console.log("Now, `map` will be filled with random entries.");
+
+    let iterationCounter = 0;
+    while (true) {
+        const randomKey = Math.floor(Math.random() * 100000);
+        await map.put('key' + randomKey, 'value' + randomKey);
+        await map.get('key' + randomKey);
+        if (++iterationCounter === 10) {
+            iterationCounter = 0;
+            const size = await map.size();
+            console.log(`Current map size: ${size}`);
+        }
+    }
+}
+
 (async () => {
     try {
-        const client = await Client.newHazelcastClient(createClientConfig());
+        const client = await Client.newHazelcastClient(
+            {
+                network: {
+                    hazelcastCloud: {
+                        discoveryToken: 'YOUR_CLUSTER_DISCOVERY_TOKEN'
+                    },
+                    ssl: {
+                        enabled: true,
+                        sslOptions: {
+                            ca: [fs.readFileSync(path.resolve(path.join(__dirname, 'ca.pem')))],
+                            cert: [fs.readFileSync(path.resolve(path.join(__dirname, 'cert.pem')))],
+                            key: [fs.readFileSync(path.resolve(path.join(__dirname, 'key.pem')))],
+                            passphrase: 'YOUR_SSL_PASSWORD',
+                            rejectUnauthorized: false
+                        }
+                    }
+                },
+                clusterName: 'YOUR_CLUSTER_NAME',
+                properties: {
+                    'hazelcast.client.cloud.url': 'YOUR_DISCOVERY_URL',
+                    'hazelcast.client.statistics.enabled': true,
+                    'hazelcast.client.statistics.period.seconds': 1,
+                }
+            }
+        );
         const map = await client.getMap('map');
```

Comment:
Let's use exactly the same map name: 'cities'. And can you move getMap call in mapExample? Let's match Java 1 to 1.